src/installation.md

```diff
@@ -51,4 +51,6 @@ sudo reboot
 
 ## Step 8: Launched Sugar
 
-Once the Raspberry Pi reboots, Sugar should start automatically. If not you may have to follow up with troubleshoots.
\ No newline at end of file
+Once the Raspberry Pi reboots, Sugar should start automatically. 
+Dont Forget to check if the Sugar Desktop Environment is installed , and turning off auto logging in RPi-config settings. 
+If not you face any difficulties you may have to follow up with troubleshoots.
```

Comment:
Not sure what you are saying here. Did you mean:

If you have any difficulties with installation, please go to the troubleshooting guide.

Reply:
just troubleshoot option available in VM